src/go/rpk/pkg/cli/acl/acl.go

```diff
@@ -72,17 +72,34 @@ group.
 
 PRINCIPALS
 
-All ACLs require a principal. A principal is composed of two parts: the type
-and the name. Within Redpanda, only one type is supported, "User". The reason
-for the prefix is that a potential future authorizer may add support for
-authorizing by Group or anything else.
+All ACLs require a principal or a role. A principal comprises a user and type.
+Within Redpanda, only the "User" type is supported. The "User" prefix ensures
```

Comment:
Why is "User" init cap here? It isn't above. Is the syntax for specifying a user case-sensitive? If not, no need for initcap here/throughout.

Reply:
@oleiman In the UI/CLI, are types always initcap?

Reply:
In the command `rpk acl create --allow-principal "User:foo" ...` the "User" prefix is indeed case sensitive (initcap). When we talk about users in the abstract, it's not usually capitalized as far as I can tell.